docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+services:
+  frontend:
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile
+    volumes:
+     - ./frontend:/app
+     - /app/node_modules
+    ports:
+      - 3000:3000
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+  ts-backend:
+    build:
+      context: ./backend/typescript
+      dockerfile: Dockerfile
+    volumes:
+      - ./backend/typescript:/app
+      - /app/node_modules
+    ports:
+      - 5000:5000
+    dns:
+      - 8.8.8.8
+    depends_on:
+      - db
+  py-backend:
+    restart: always
+    build:
+      context: ./backend/python
+      dockerfile: Dockerfile
+    ports:
+      - 5000:5000
+    volumes:
+     - ./backend/python:/app
+    depends_on:
+      - db
+  db:
+    image: postgres:12-alpine
+    ports:
+      - 5432:5432
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    env_file:
+      - ./backend/typescript/.env
```

Comment:
Should we set an `env_file` for the other services as well?

Reply:
Switched to using a global `.env` file shared by all services (organized by prefixes), I'll leave it to individual teams to decide on how they'll manage secrets